src/components/slide/content/poster/poster-single.jsx

```diff
@@ -33,9 +33,22 @@ function PosterSingle({ configurationChange, feedSource, configuration }) {
   const [singleSelectedOccurrence, setSingleSelectedOccurrence] =
     useState(null);
 
-  const searchEndpoint = feedSource.admin[0].endpointSearch ?? null;
-  const optionsEndpoint = feedSource.admin[0].endpointOption ?? null;
-  const entityEndpoint = feedSource.admin[0].endpointEntity ?? null;
+  const {
```

Comment:
This can be done directly in the `function PosterSingle({ ...blabla....` but this is fine to